geomesa-hbase/geomesa-hbase-datastore/src/main/java/org/locationtech/geomesa/hbase/filters/JSimpleFeatureFilter.java

```diff
@@ -131,15 +161,20 @@ private int getLen(String s) {
         int filterLen = Bytes.readAsInt(pbBytes, sftLen + 4, 4);
         String filterString = new String(Bytes.copy(pbBytes, sftLen + 8, filterLen));
 
+        int transformLen = Bytes.readAsInt(pbBytes, sftLen + filterLen + 8, 4);
+        String transformString = new String(Bytes.copy(pbBytes, sftLen + filterLen + 12, transformLen));
```

Comment:
these could be optimized by using the `new String(byte[] bytes, int offset, int length)` constructor